libamqpprox/amqpprox_limitcontrolcommand.cpp

```diff
@@ -29,189 +32,201 @@ namespace amqpprox {
-                              ->toString()
-                       << " in alarm only mode.\n";
-                output << "The limiter will only log at warning level with "
-                          "AMQPPROX_CONNECTION_LIMIT as a substring and the "
-                          "relevant limit details, when the new incoming "
-                          "connection violates the specified limit.\n";
-            }
-            else {
-                output << "For vhost " << vhostName << ", "
-                       << connectionLimiterManager
-                              ->addConnectionRateLimiter(vhostName,
-                                                         numberOfConnections)
-                              ->toString()
-                       << "\n";
-            }
-        }
-        else if (subcommand == "DEFAULT") {
-            uint32_t numberOfConnections;
-            if (!(iss >> numberOfConnections && numberOfConnections >= 0 &&
-                  numberOfConnections <
-                      std::numeric_limits<uint32_t>::max())) {
-                output << "Invalid numberOfConnections provided.\n";
-                return;
-            }
-
-            if (alarmOnly) {
-                connectionLimiterManager
-                    ->setAlarmOnlyDefaultConnectionRateLimit(
-                        numberOfConnections);
-                output << "Default connection rate limit is set to "
-                       << connectionLimiterManager
-                              ->getAlarmOnlyDefaultConnectionRateLimit()
-                       << " connections per second in alarm only mode.\n";
-                output << "The limiter will only log at warning level with "
-                          "AMQPPROX_CONNECTION_LIMIT as a substring and the "
-                          "relevant limit details, when the new incoming "
-                          "connection violates the default limit for all "
-                          "vhosts.\n";
-            }
-            else {
-                connectionLimiterManager->setDefaultConnectionRateLimit(
-                    numberOfConnections);
-                output << "Default connection rate limit is set to "
-                       << connectionLimiterManager
-                              ->getDefaultConnectionRateLimit()
-                       << " connections per second.\n";
-            }
+    if (isDisable) {
+        if (isDefault) {
+            connectionLimiterManager
+                ->removeAlarmOnlyDefaultConnectionRateLimit();
+            output << "Successfully disabled default alarm only "
+                      "connection rate limit\n ";
         }
         else {
-            if (alarmOnly) {
-                output << "Invalid subcommand provided for LIMIT "
-                          "CONN_RATE_ALARM command.\n";
-            }
-            else {
-                output << "Invalid subcommand provided for LIMIT CONN_RATE "
```

Comment:
Some of the error messages are a bit less specific now, but should still point to what failed to parse